src/styles/document-style.css

```diff
@@ -215,18 +230,42 @@ p {
 	.social__handles {
 		padding-right: 0;
 	}
-}
 
-@media only screen and (max-width: 500px) {
-	:root {
-		font-size: 62%;
+	.nav {
+		display: none;
 	}
 
-	body > .header,
-	.main {
-		justify-content: center;
-		padding: 10px;
-		gap: 10px;
+	.hamburger_icon {
+		display: block;
+		color: var(--COLOR-TXT);
+		font-size: 3.4rem;
+	}
+
+	.show {
+		backdrop-filter: blur(10px);
+		position: relative;
+	}
+
+	.show .main {
+		display: none;
+	}
+
+	.show .nav {
+		display: flex;
+		z-index: 88;
```

Comment:
Suggestion:  use variables for z-index in case of big projects it will help

Reply:
Done